WorldTrotter/WorldTrotter/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  WorldTrotter
+//
+//  Created by Diane Skelskey on 12/23/21.
```

Comment:
Change this to Ian Skelskey

Reply:
Will have to change... name in Xcode? Make new user on mombook